addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -4,6 +4,7 @@
 from odoo.exceptions import AccessError, UserError, ValidationError
 from odoo.osv import expression
 
+
```

Comment:
unwanted new line

Reply:
It's wanted, python lint/formatter should have 2 empty lines above class